### DISCUSSION TITLE
When QUIC Server start a new stream, QUIC Client‘s connection callback  did not receive the event  PEER_STREAM_STARTED

### DISCUSSION THREAD

**Discussion by Ang0426** [microsoft/msquic#3502] [Q&A]
QUIC Client started a new connection, and the connection is connected.
Then I try to use the QUIC Server start a new stream on the connection, it succeed when call StreamOpen, and when call StreamStart, it always return 0x0x703e5(Status), QUIC Client's connection callback did not receive any event, and the buffer is can not send.

In the sample.c ，QUIC client did the StreamOpen and StreamStart, it is OK, and QUIC server use the same Stream to send data.

I tried change the StreamStart‘s ’parameter to QUIC_STREAM_START_FLAG_INDICATE_PEER_ACCEPT, but it didn‘t work.
Do I need do someting  to reslove it , thank you.

**Comment by nibanks** [comment]
Make sure the peer configured their settings to allow enough streams. Just because the server calls `StreamStart` doesn't mean it can send the stream to the client (yet). If the client hasn't been configured to accept any streams, the stream will just be queued (forever) on the server side.

**Reply by Ang0426** [reply]
Yes ，it is my fault, now it can start a stream , thanks you .
